homeassistant/components/broadlink/switch.py

```diff
@@ -34,7 +34,9 @@
 
 DEFAULT_NAME = "Broadlink switch"
 DEFAULT_TIMEOUT = 10
+DEFAULT_RETRY = 3
```

Comment:
Change it to 2 to keep the current behavior

Reply:
done.